webapp/packages/core/src/blocks/InFocus.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {
+  useEffect, useRef, ReactElement, MutableRefObject
+} from 'react';
+
+type InFocusProps = {
+  children: ReactElement;
+}
+
+export function InFocus({ children }: InFocusProps) {
+  const isFirstRender: MutableRefObject<boolean> = useRef(true);
+  const childRef: MutableRefObject<HTMLElement | null> = useRef(null);
+
+  useEffect(() => {
```

Comment:
useLayoutEffect better for purpose like this

Reply:
fixed